test/unittests/libethereum/ValidationSchemes.cpp

```diff
@@ -0,0 +1,216 @@
+/*
+    This file is part of cpp-ethereum.
+
+    cpp-ethereum is free software: you can redistribute it and/or modify
+    it under the terms of the GNU General Public License as published by
+    the Free Software Foundation, either version 3 of the License, or
+    (at your option) any later version.
+
+    cpp-ethereum is distributed in the hope that it will be useful,
+    but WITHOUT ANY WARRANTY; without even the implied warranty of
+    MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+    GNU General Public License for more details.
+
+    You should have received a copy of the GNU General Public License
+    along with cpp-ethereum.  If not, see <http://www.gnu.org/licenses/>.
+*/
+/** @file ValidationSchemes.cpp
+ * @author Dimitry Khokhlov <dimitry@ethereum.org>
+ * @date 2019
+ * Validation scheme unit tests.
+ */
+
+#include <libethereum/ValidationSchemes.h>
+#include <test/tools/libtesteth/JsonSpiritHeaders.h>
+#include <test/tools/libtesteth/TestHelper.h>
+
+using namespace std;
+using namespace dev;
+using namespace dev::eth;
+using namespace dev::test;
+namespace js = json_spirit;
+
+BOOST_FIXTURE_TEST_SUITE(ValidationSchemes, TestOutputHelperFixture)
```

Comment:
These tests must go to `aleth-unittests`

Reply:
Where is that?

Reply:
1) why aleth-unittests ? 
2) how alet-unittest run? is it set to autorun on travis? 
3) to add it to aleth I have to exclude that file from unittests and rewrite it using GTest suite.

Reply:
do you plan to switch all unittests from testeth to aleth-unittest using gtest only ?

Reply:
Only true unit tests: the ones that do not read JSON files.